extern/storage-sealing/commit_batch.go

```diff
@@ -285,7 +286,14 @@ func (b *CommitBatcher) processBatch(cfg sealiface.Config) ([]sealiface.CommitBa
 		return []sealiface.CommitBatchRes{res}, xerrors.Errorf("couldn't get miner info: %w", err)
 	}
 
-	goodFunds := big.Add(b.feeCfg.MaxCommitGasFee, collateral)
+	bf, err := b.api.ChainBaseFee(b.mctx, tok)
+	if err != nil {
+		return []sealiface.CommitBatchRes{res}, xerrors.Errorf("couldn't get base fee: %w", err)
+	}
+
+	aggFee := miner5.AggregateNetworkFee(len(infos), bf)
+
+	goodFunds := big.Add(b.feeCfg.MaxCommitGasFee, big.Add(collateral, aggFee))
```

Comment:
Hm.... shouldn't we have a new field for this? Or shouldn't the max fee be a function of the number of sectors, or something like that?

Reply:
https://github.com/filecoin-project/lotus/pull/6420 does that